vmaas_sync/send_messages.go

```diff
@@ -53,7 +53,6 @@ func sendMessages(ctx context.Context, inventoryIDs ...string) {
 	events := make([]mqueue.PlatformEvent, len(inventoryIDs))
 
 	for i, id := range inventoryIDs {
-		utils.Log("inventoryID", id).Debug("Sending evaluation kafka message")
```

Comment:
Are you sure the long duration was caused by logging? It could be caused by sending messages through the Kafka. Anyway we can set INFO logging level in prod and so this message will be disabled.

Reply:
Ok, merging. We can simply revert it later if we need.